run.py

```diff
@@ -71,31 +71,36 @@ def get_generator_cmd(simulator, simulator_yaml, cov, exp, debug_cmd):
   """ Setup the compile and simulation command for the generator
 
   Args:
-    simulator      : RTL simulator used to run instruction generator
-    simulator_yaml : RTL simulator configuration file in YAML format
+    simulator      : RTL/pyflow simulator used to run instruction generator
+    simulator_yaml : RTL/pyflow simulator configuration file in YAML format
     cov            : Enable functional coverage
     exp            : Use experimental version
     debug_cmd      : Produce the debug cmd log without running
 
   Returns:
-    compile_cmd    : RTL simulator command to compile the instruction generator
-    sim_cmd        : RTL simulator command to run the instruction generator
+    compile_cmd    : RTL/pyflow simulator command to compile the instruction
+    generator
```

Comment:
can you indent `generator` to match indentation with `RTL/pyflow` ? both here and below